types/stripe-js/checkout.d.ts

```diff
@@ -448,7 +444,7 @@ export interface StripeCheckout {
   }) => Promise<StripeCheckoutUpdateLineItemQuantityResult>;
   updateShippingOption: (
     shippingOption: string
-  ) => Promise<StripeCheckoutResult>;
+  ) => Promise<StripeCheckoutUpdateShippingOptionResult>;
```

Comment:
It's added here already https://github.com/stripe/stripe-js/blob/c057b507f5f9ac01e167c4f9683b61010cd18ea2/types/stripe-js/checkout.d.ts#L409-L411